test/e2e/libpod_suite_test.go

```diff
@@ -460,3 +460,36 @@ func (p *PodmanTest) BuildImage(dockerfile, imageName string) {
 	session.Wait(120)
 	Expect(session.ExitCode()).To(Equal(0))
 }
+
+//GetHostDistribution returns the dist in string format. If the
+//distribution cannot be determined, an empty string will be returned.
+//Possible returns must be RHEL, Ubuntu, CentOS, Fedora
+func (p *PodmanTest) GetHostDistribution() string {
+	var distribution string
+	content, err := ioutil.ReadFile("/etc/os-release")
+	if err != nil {
+		return ""
+	}
+	for _, line := range content {
+		if strings.HasPrefix(fmt.Sprintf("%s", line), "NAME") {
```

Comment:
I would generally look at `ID` and not name.

Reply:
i like that better ... thanks i'll use the ID and it eliminates the switch